### ISSUE TITLE
Merge NootRX into NootedRed

### ISSUE THREAD

**Issue by VisualEhrmanntraut** [ChefKissInc/NootedRed#322]
### macOS Version

None

### What is your CPU's model?

N/A

### Please describe the feature in detail.

N/A

### What alternatives have you tried, if any?

N/A

**Comment by VisualEhrmanntraut** [comment]
<img width="1044" alt="image" src="https://github.com/user-attachments/assets/ca33dd05-887e-441c-a04a-e86ebf5e6c3e">

**Comment by VisualEhrmanntraut** [comment]
<img width="268" height="617" alt="Image" src="https://github.com/user-attachments/assets/b02f7f8b-97c2-4f8a-851c-30f87957c3f5" />

**Comment by keybordguy** [comment]
Any updates on this and adding rdna 2 igpu support since i couldnt comment on there?

**Comment by keybordguy** [comment]
> Any news..!? This feature would be the top of the top 😍

i had the same question for 4 months and my guess is that the developers of this kext are trying to make it PERFECT instead of adding a bunch of features and all of them are buggy.

**Comment by VisualEhrmanntraut** [comment]
> > Any news..!? This feature would be the top of the top 😍
> 
> i had the same question for 4 months and my guess is that the developers of this kext are trying to make it PERFECT instead of adding a bunch of features and all of them are buggy.

one developer (me), and nah, I'm rather working on my Apple Silicon (ARM Mac/iPhone/etc) emulator (https://github.com/ChefKissInc/Inferno)

**Comment by Adonis4U** [comment]
> > > Any news..!? This feature would be the top of the top 😍
> > 
> > i had the same question for 4 months and my guess is that the developers of this kext are trying to make it PERFECT instead of adding a bunch of features and all of them are buggy.
> 
> one developer (me), and nah, I'm rather working on my Apple Silicon (ARM Mac/iPhone/etc) emulator (https://github.com/ChefKissInc/Inferno)

Yes, certainly that emulator project of yours is fantastic! Really! But I really hoped even being you the only developer (and I thought there was someone to support you in this majestic work you did) that you would have made NootedRed even more powerful and full support for other AMD cards etc etc ... In short, it's a shame to leave such a majestic project 🥵 You have done a job like a real Chef! And I think the whole world is grateful to you for that! So even this feature that I was seeing you were working on would be the top (being that I saw that you were working on it until several months ago). Too bad 😭 really. I'm so sorry for that.

**Comment by VisualEhrmanntraut** [comment]
well, part of the reason for the lack of work on the hackintosh projects is lack of funding. also, I never said anything is cancelled or abandoned, take it as a hiatus for now.

**Comment by VisualEhrmanntraut** [comment]
plan to get it ready this year #semi-soon

**Comment by VisualEhrmanntraut** [comment]
> > Any news..!? This feature would be the top of the top 😍
> 
> i had the same question for 4 months and my guess is that the developers of this kext are trying to make it PERFECT instead of adding a bunch of features and all of them are buggy.

I forgot to say, that I ensure the quality of the kext regardless of feature or development time; I have a quality control system with a few volunteers that test changes alongside me before they are committed into the repository.

**Comment by VisualEhrmanntraut** [comment]
The removal of the model mechanism in the latest release is part of this work btw.
